src/coreclr/src/debug/dbgutil/elfreader.h

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <windows.h>
```

Comment:
Do we need all these windows and .Net specific headers here? The definition of the class only appears to use primitive types and some C++ standard library types.

Reply:
Need windows.h so we can include cordebug.h etc for the ICorDebugDataTarget interface.

Reply:
Can you put that in the .cpp instead of the .h? For work that is intended to be xplat I assume it is helpful if we don't propogate OS specific headers any farther than we have to.